api/src/main/java/org/pmiops/workbench/db/model/DbUser.java

```diff
@@ -109,6 +109,7 @@ public enum DbPartnerDiscoverySource {
     IGNITE_NORTHWELL_HEALTH_FEINSTEIN_INSTITUTE_FOR_MEDICAL_RESEARCH,
     NATIONAL_ALLIANCE_FOR_HISPANIC_HEALTH_NAHH,
     NATIONAL_BAPTIST_CONVENTION_USA_INC,
+    NATIONAL_LATINO_NETWORK_FOR_PRECISION_MEDICINE_AND_HEALTH_DISPARITIES,
```

Comment:
Why would we define this enum in DbPartnerDiscoverySource and also in workbench-api.yaml(Which generates a class called PartnerDiscoverySource)? These enums are exactly the same and you have to keep them in sync. Seems like we could just keep the one defined in the workbench-api.yaml and reference that in the codebase.

Reply:
Good point! I removed the enum from DbUser. Thanks for pointing it out.